mail/chrome/messenger/messenger.properties

```diff
@@ -234,7 +234,8 @@ remoteContentPrefAccesskey=O
 remoteContentPrefLabelUnix=Preferencje
 remoteContentPrefAccesskeyUnix=P
 
-remoteAllow=Zezwalaj na zdalną zawartość z %S
+remoteAllowResource=Zezwalaj na zdalną zawartość z %S
+remoteAllowAll=Zezwalaj na zdalną zawartość z #1 źródła wymienionego powyżej;Zezwalaj na zdalną zawartość ze wszystkich #1 źródeł wymienionych powyżej;Zezwalaj na zdalną zawartość ze wszystkich #1 źródeł wymienionych powyżej
```

Comment:
Do you think that "wszystkich" is really needed here when we have the number of URLs?